src/VerseCheck/EditVerseArea/index.js

```diff
@@ -97,7 +97,7 @@ const EditVerseArea = ({
     />,
   );
   const checkBoxText = isVerseChanged ? translate('next_change_reason') : translate('first_make_change');
-  const fontClass = getFontClassName(targetLanguageFont);
+  const fontClass = getFontClassName(targetLanguageFont) || 'default-text'; // TRICKY defaulting to the 'default-text' class prevents 'form-control' class from resetting font-size to 14px
```

Comment:
default-text???? Where's that coming from? Never heard of it? 
I used default and or just an empty string.

Reply:
Added a new class to styles.css: https://github.com/unfoldingWord/translationCore/pull/7057/files#diff-84ffa77d0c220127634fcb67237c716e

Reply:
LGTM